.github/workflows/pr_scan.yml

```diff
@@ -13,10 +13,11 @@ jobs:
   build:
     env:
       sonarToken: ${{ secrets.SONAR_TOKEN }}
-      destination: "name=iPhone 15 Pro,OS=17.2"
+      destination: "name=iPhone 15 Pro,OS=18.1"
+      destination_iPad: "name=iPad (10th generation),OS=18.1"
```

Comment:
```suggestion
      destination_iPad: "name=iPad Air 11-inch (M3),OS=18.4"
```

Reply:
Did you just pick the latest? I try to stay conservative upgrading to only Xcode 16.1 and I don't think it has 18.4